app/assets/stylesheets/cart.css.scss

```diff
@@ -30,11 +30,12 @@
 }
 
 #cart-table {
-    margin-left:50px;
-    width: 850px;
-    border: none;
-    border-radius: 5px;
-    box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+  margin-left:50px;
+  width: 850px;
+  border: solid 1px #696969;
+  border-radius: 5px 5px 5px 5px;
+  box-shadow: 0px 10px 20px rgba(0,0,0,.3);
```

Comment:
`.3` should be written with a leading zero as `0.3`<br>Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`